test/parallel/test-metrics-meter.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const common = require('../common');
+
+const assert = require('assert');
+const { subscribe } = require('node:diagnostics_channel');
+const { meter, Meter, Metric, MetricReport } = require('node:metrics');
+
+const testMeter = meter('test', 100, { base: 'test' });
+assert.ok(testMeter instanceof Meter);
+assert.strictEqual(testMeter.value, 0);
+assert.strictEqual(testMeter.interval, 100);
+assert.ok(testMeter.metric instanceof Metric);
+
+const { metric } = testMeter;
+assert.strictEqual(metric.type, 'meter');
+assert.strictEqual(metric.name, 'test');
+assert.deepStrictEqual(metric.meta, { base: 'test' });
+assert.strictEqual(metric.channelName, 'metrics:meter:test');
+
+const messages = [
+  [1, { base: 'test' }],
+  [124, { base: 'test', meta: 'extra' }],
```

Comment:
Is it expected that the value of the meta `{ base: 'test', meta: 'extra' }` is `124`, while it was only `mark`-ed with `123`?

Reply:
Yes, it's cumulative within the specified window.

Reply:
The meta `{ base: 'test', meta: 'extra' }` is only marked once with value `123` though.

Reply:
The value is not unique to the meta, it is unique to the instance. That single data point report just has additional data on it. You can have type-level meta and you can have add report-level meta onto individual reports, but the value is always based on the type instance.

Reply:
> In the case of the meta folding, I'm thinking of instead of having one meta in the final reporting, keeping the instance and report metas separate so the consumers can make their own decisions on what to do with the different metadata sets.

From the current PR status, the aggregation is already done inside the `Meter.mark`. Is this describing the goal of the API, rather than the current PR status?

Reply:
Yes, that's ideation on a possible improvement to make the distinction between the two meta stages and to allow taking the data from one and not the other if a particular format doesn't support it.